spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/notify/MicrosoftTeamsNotifier.java

```diff
@@ -0,0 +1,372 @@
+    public URI getImageUrl() {
+        return imageUrl;
+    }
+
+    public void setImageUrl(URI imageUrl) {
+        this.imageUrl = imageUrl;
+    }
+
+    public void setThemeColor(String themeColor) {
+        this.themeColor = themeColor;
+    }
+
+    public String getDeregisterActivitySubtitlePattern() {
+        return deregisterActivitySubtitlePattern;
+    }
+
+    public void setDeregisterActivitySubtitlePattern(String deregisterActivitySubtitlePattern) {
+        this.deregisterActivitySubtitlePattern = deregisterActivitySubtitlePattern;
+    }
+
+    public String getRegisterActivitySubtitlePattern() {
+        return registerActivitySubtitlePattern;
+    }
+
+    public void setRegisterActivitySubtitlePattern(String registerActivitySubtitlePattern) {
+        this.registerActivitySubtitlePattern = registerActivitySubtitlePattern;
+    }
+
+    public String getStatusActivitySubtitlePattern() {
+        return statusActivitySubtitlePattern;
+    }
+
+    public void setStatusActivitySubtitlePattern(String statusActivitySubtitlePattern) {
+        this.statusActivitySubtitlePattern = statusActivitySubtitlePattern;
+    }
+
+    public String getDeRegisteredTitle() {
+        return deRegisteredTitle;
+    }
+
+    public void setDeRegisteredTitle(String deRegisteredTitle) {
+        this.deRegisteredTitle = deRegisteredTitle;
+    }
+
+    public String getRegisteredTitle() {
+        return registeredTitle;
+    }
+
+    public void setRegisteredTitle(String registeredTitle) {
+        this.registeredTitle = registeredTitle;
+    }
+
+    public String getStatusChangedTitle() {
+        return statusChangedTitle;
+    }
+
+    public void setStatusChangedTitle(String statusChangedTitle) {
+        this.statusChangedTitle = statusChangedTitle;
+    }
+
+    public String getMessageSummary() {
+        return messageSummary;
+    }
+
+    public void setMessageSummary(String messageSummary) {
+        this.messageSummary = messageSummary;
+    }
+
+
+    @JsonSerialize
+    class Message {
+
+        private String summary;
+        private String themeColor;
+        private String title;
+        private List<Section> sections = new ArrayList<>();
+
+        public String getSummary() {
+            return summary;
+        }
+
+        public void setSummary(String summary) {
+            this.summary = summary;
+        }
+
+        public String getTitle() {
+            return title;
+        }
+
+        public String getThemeColor() {
+            return themeColor;
+        }
+
+        public void setSections(List<Section> sections) {
+            this.sections = sections;
+        }
+
+        public void setTitle(String title) {
+            this.title = title;
+        }
+
+        public void setThemeColor(String themeColor) {
+            this.themeColor = themeColor;
+        }
+
+        public List<Section> getSections() {
+            return sections;
+        }
+
+    }
+
+    @JsonSerialize
+    class Section {
```

Comment:
either don't make it an inner class or make it static

Reply:
Addressed in [95d0b934cb83ed50f29baefc70f583f901db4c87](https://github.com/codecentric/spring-boot-admin/pull/592/commits/95d0b934cb83ed50f29baefc70f583f901db4c87)